solutions/src/main/java/ru/sberbank/school/task02/Main.java

```diff
@@ -0,0 +1,39 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+
+/**
+ * Created by Mart
+ * 05.05.2019
+ **/
+public class Main {
+    public static void main(String[] args) {
+        ServiceFactory serviceFactory = new ServiceFactoryImpl();
+        ExternalQuotesService externalQuotesService = new ExternalQuotesServiceDemo();
+        for (Quote q : externalQuotesService.getQuotes(Symbol.USD_RUB)) {
+            System.out.println("offer: " + q.getOffer() +  " bid " + q.getBid()+"  Volume: " + q.getVolumeSize());
```

Comment:
WhitespaceAround: &#39;+&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a450022880>*